src/components/PostEditor/LinkPreview/LinkPreview.scss

```diff
@@ -1,31 +1,35 @@
-.link-preview {
-  display: flex;
-  padding: $space-5x;
+$preview-image-dimension: $space-20x;
```

Comment:
Inherits all style from `components/LinkPreview` except where extra is needed for `PostEditor` interaction.